style.css

```diff
@@ -514,8 +569,112 @@ h4 {
   display: flex;
   justify-content: center;
   align-items: center;
-  width: 77px;
+  width: 76px;
   height: 40px;
   background: #fff;
   border-radius: 8px;
 }
+
+.form {
+  background-image: url('./images/form/illustration-contact-form.svg');
+  background-repeat: no-repeat;
+  background-size: 40px;
+  background-position: right bottom;
+  padding: 0;
+  margin-top: -850px;
+}
+
+.form-text {
+  grid-column: 1/13;
+  display: flex;
+  justify-content: center;
+  margin-left: 11px;
+  margin-bottom: 10px;
+}
+
+.form-text h3 {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  font-weight: 600;
+  line-height: 52px;
+  text-align: center;
+  margin-bottom: 35px;
+  color: #172b4d;
+}
+
+.inputs {
+  grid-column: 1/13;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.input {
+  width: 327px;
+  height: 40px;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  background: #fff;
+  padding: 0;
+  margin-bottom: 30px;
+  margin-left: 15px;
+}
+
+.msg {
+  width: 327px;
+  height: 114px;
+  font-weight: 500;
+  border: 1px solid#60C095;
+  border-radius: 4px;
+  background: #fff;
+  padding: 0;
+  margin-bottom: 30px;
+  margin-left: 15px;
+}
+
+.email-address {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 20px;
+  font-weight: 300;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  margin-bottom: 100px;
+}
+
+.form-bttn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 4px;
+  width: 122px;
+  height: 48px;
+  background: #36b37f;
+  border: 0;
+  margin-left: 110px;
+  margin-right: 105px;
+  margin-top: 0;
+  margin-bottom: 0;
+  color: white;
+  transition: all 0.3s;
+  font-size: 18px;
+}
+
+.input:hover {
+  transition: all 0.3s;
+  background: #c8d5f5;
+}
+
+.msg:hover {
+  transition: all 0.3s;
+  background: #dfe1e6;
+}
+
+.form-bttn:hover {
+  border: 1px solid rgb(2, 155, 35);
+  color: #6588e2;
+  background-color: #c1c7d0;
+}
```

Comment:
- [ ] Please as requested by the previous code reviewer, interactions such as hover, active (when it's pressed), disabled and enable (default state) should be styled based on the design you selected as shown in the Figma template. Kindly work on it
  
  ### Your design
  ![Screenshot from 2023-02-21 16-10-19](https://user-images.githubusercontent.com/92303891/220402038-75d6dc30-cbd7-42f9-bb48-4a09c7bdc8d8.png)
  
  ### Figma design
  ![image](https://user-images.githubusercontent.com/92303891/220402942-bc1144cc-5c1b-420b-83cd-ed6ca9c5fbdd.png)